lib/dealog_backoffice_web/views/error_helpers.ex

```diff
@@ -22,7 +22,13 @@ defmodule DealogBackofficeWeb.ErrorHelpers do
   Check if the given form field has errors.
   """
   def has_errors?(form, field) do
-    Map.has_key?(Map.from_struct(form).errors, field)
+    # TODO Consolidate form error processing
```

Comment:
This should get addressed ASAP. Check how to use schemaless changesets for the message handling.

Reply:
DEV-136